### PR TITLE
Add drawing rectangles module

### PR DESCRIPTION
**Pull request by Mridul97**
Issue #430 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Mridul97** [comment]
@tech4GT Please have a look at it!

Output :- 

ss_35

**Comment by HarshKhandeparkar** [comment]
@Mridul97 how about thickness of border input?

**Comment by HarshKhandeparkar** [comment]
Ok 👍 NP.

**Comment by Mridul97** [comment]
Thanks for the suggestions! I will implement those. pixels.shape stores the shape of the pixels array and pixels.data is an array containing the information of the pixels!

**Comment by HarshKhandeparkar** [comment]
I found docs about the pixels ndarray. Thanks for your help!

File: src/Modules.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ module.exports = {
   'convolution': require('./modules/Convolution'),
   'crop': require('./modules/Crop'),
   'decode-qr': require('./modules/DecodeQr'),
+  'drawing-rectangles': require('./modules/DrawingRectangles'),
```
Comment (jywarren) [inline src/Modules.js]: Could we call it `draw-rectangle`?

**Comment by jywarren** [comment]
awesome. Ready to go, then? Thanks!!

**Comment by Mridul97** [comment]
Yes, I think so! I have rebased the branch! will open a issue for adding features of border thickness and border color. Thanks!

**Pull request merged**